appinventor/appengine/war/Ya.css

```diff
@@ -990,6 +990,65 @@ select.ode-PropertyEditor[disabled] {
   border: 2px solid black;
 }
 
+
+.ode-SimpleMockFormPhoneLandscape {
+    background: url(images/phoneLandscape.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 100% 100%;
+    /*padding: 5px 54px 5px 54px;*/
+    /*padding: 11px 54px 11px 64px;*/
+    padding: 11px 43px 7px 48px;
+}
+
+.ode-SimpleMockFormPhonePortrait {
+  background: url(images/phonePortrait.png);
+  /*   background: url(images/phonePortrait.png); */
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: 100% 100%;
+  /*padding: 50px 11px 64px 11px;*/
+  padding: 50px 11px 43px 7px;
+}
+
+.ode-SimpleMockFormPhoneLandscapeTablet {
+    background: url(images/phoneLandscapeTablet.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 100% 100%;
+    /*padding: 5px 55px 5px 54px;;*/
+    /*padding: 11px 54px 11px 64px;*/
+    padding: 15px 56px 12px 64px;;
+}
+.ode-SimpleMockFormPhonePortraitTablet {
+  background: url(images/phonePortraitTablet.png);
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: 100% 100%;
+  /*padding: 54px 11px 64px 11px;*/
+  padding: 64px 13px 55px 13px;
+}
+/**/
+.ode-SimpleMockFormPhoneLandscapeMonitor {
+  background: url(images/phoneLandscapeMonitor.png);
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: 100% 100%;
+  /*padding: 5px 54px 5px 54px;*/
+  /*padding: 11px 54px 11px 64px;*/
+  padding: 44px 43px 7px 48px;
+}
+
+.ode-SimpleMockFormPhonePortraitMonitor {
+  background: url(images/phonePortraitMonitor.png);
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: 100% 100%;
+  /*padding: 5px 55px 5px 54px;;*/
+  /*padding: 11px 54px 11px 64px;*/
+  padding: 99px 36px 75px 34px;
+}
+/**/
```

Comment:
What are the purposes of these empty comments?

Reply:
Added start / end comment-tag to show code-updates for phone skins